packages/react-layouts/src/Modal/index.test.js

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+import React from 'react';
+import { mount } from 'enzyme';
+import Modal from './index';
+import Header from './Header';
+import { HeaderIcon } from './Header';
+import ActionBar from './ActionBar';
+
+describe('Modal', () => {
+  it('renders the expected markup', () => {
+    const wrapper = mount(
+      <Modal contentLabel="Foobar" isOpen>
+        <span>Hello, World!</span>
+      </Modal>
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders the expected markup', () => {
+    const wrapper = mount(
+      <Modal
+        title="Did You Know...?"
+        icon="question"
+        importance="info"
+        renderActionLeft={() => 'foobar'}
+        renderActionRight={() => 'foobar'}
+        isOpen
+      >
+        <span>Hello, World!</span>
+      </Modal>
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Could we narrow down this snapshot test to multiple small snapshot tests that check specifically for `title`, `icon`, `importance` `renderActionLeft` etc?